README.md

```diff
@@ -846,6 +846,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [An API of Ice And Fire](https://anapioficeandfire.com/) | Game Of Thrones API | No | Yes | Unknown |
+| [Breaking Bad API](https://breakingbadapi.com/) | Breaking Bad API| No | Yes | Unknown |
```

Comment:
Please link to https://breakingbadapi.com/documentation.

And please add a space after the description ` Breaking Bad API `

Reply:
These corrections should be made now! Thank you for taking the time to view my pr!